ops/ansible/roles/postmaster_deploy/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+- name: Install main list of packages
+  apt: pkg={{item}} state=installed update_cache=true
+  sudo: yes
+  with_items:
+    - git
+    - python
+    - python-pip
+    - python-dev
+    - python-virtualenv
+    - libldap2-dev
+    - libssl-dev
+    - libsasl2-dev
+    - libffi-dev
+    - apache2
+    - libapache2-mod-wsgi
+    - libmysqlclient-dev
+  when: provision_type == "dev"
```

Comment:
Now this will only run when Vagrant calls it. I believe what we want is to install everything but MySQL for every provision type. Then for Vagrant, we want to install MySQL on top of that.